person.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'nameable'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', parent_permision: true)
+    super()
+    @id = 0
+    @name = name
+    @age = age
+    @parent_permision = parent_permision
+    @rentals = []
+  end
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
```

Comment:
- [ Optional ] I think it would be nice if you put the `readers` and `accessors` before the `initialize` method for clarity purposes, I almost missed them when going through this code. I think your code will be much clearer if you place them at the top of the class the way you've done for other classes.